setup.py

```diff
@@ -0,0 +1,14 @@
+# coding=utf-8
+from setuptools import setup
+
+setup(
+    name='http-server',
+    description='http-server',
+    version=0.1,
+    author='Joe McClenahan and Jared Scarr',
+    author_email='jmcclena94@gmail.com jaredscarr@gmail.com',
+    license='MIT',
+    py_modules=['client'],
```

Comment:
You don't include your server module in the distribution?  That's odd.  I certainly would want to.